MCprep_addon/conf.py

```diff
@@ -140,7 +153,7 @@ def icons_init(self):
 				'IMAGE')
 		except Exception as e:
 			self.log("Old verison of blender, no custom icons available")
-			self.log("\t" + str(e))
+			self.log(f"\t {e}")
```

Comment:
The `\t` isn't necessary and will most definitely cause problems with terminal output
```suggestion
			self.log(e)
```